src/views/Course.jsx

```diff
@@ -0,0 +1,17 @@
+import CoursesHero from "../components/Courses/CoursesHero";
+import CourseTextArea from "../components/Course/CourseTextArea";
+import CoreBar from "../components/CoreBar";
+import CoreFooter from "../components/CoreFooter";
+
+function Course() {
```

Comment:
You need to recieve props here from main.js

Reply:
Will need to figure out how to pull props

Reply:
You need to import useParams _from_ react-router-dom

Reply:
I figured this out, will push shortly